.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,45 @@
+name: Feature/Improvement structured request
+description: Suggest an idea for Asterisk
+title: title
+labels: feature-request
+body:
+  - type: textarea
+    id: problem
+    attributes:
+      label: Is your feature or improvement request related to a problem? Please describe.
+      value: |
+        A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+  - type: textarea
+    id: solution
+    attributes:
+      label: Describe the solution you'd like
+      value: |
+        A clear and concise description of what you want to happen.
+  - type: textarea
+    id: alternatives
+    attributes:
+      label: Describe alternatives you've considered
+      value: |
+        A clear and concise description of any alternative solutions or features you've considered.
+  - type: textarea
+    id: additional
+    attributes:
+      label: Additional context
+      value: |
+        Add any other context or screenshots about the feature request here.
+  - type: dropdown
+    id: budget
+    attributes:
+      label: How much are you willing to donate for this feature?
```

Comment:
The project does not accept donations. What is the intention of this field?

Reply:
There used to be Bounties. I think Donations sounds nicer. I know there are Jobs on the Forums, so maybe instead of budget dropdown, provide a note/link to there from GitHub ?

Reply:
Donation has different meaning with it. Having a link to the bug bounty docs page and the jobs section of the forum is fine.

Reply:
Done, thanks!